File: google/src/go/custom/custom.go
package function

import (
        "time"
        "fmt"
        "net/http"
        "log"
        "encoding/json"
)

type Message struct {
        Success bool `json:"success"`
        Payload Payload `json:"payload"`
}
        
type Payload struct {
        Test string `json:"test"`
        Time int `json:"time"`
}

func Go_custom(w http.ResponseWriter, r *http.Request) {
        start := time.Now()

        /* 
        TODO: put your code here
        You can basically do anything you want,
        but please leave the return statement header
        and the success field as it is.
        */

        elapsed := time.Since(start)
        msg := &Message{
                Success: true,
                Payload: Payload{
                    Test: "custom test",
                    Time: int(elapsed / time.Millisecond),
                },
        }
        
        b, err := json.Marshal(msg)

        if err != nil {
                log.Fatal(err)
        }

        w.Header().Set("Content-Type", "application/json")
        w.WriteHeader(200)

        fmt.Fprint(w, string(b))
}

File: azure/src/dotnet/dotnet_custom/custom.cs
using System;
using System.Net;
using System.Net.Http;
using System.IO;
using System.Diagnostics;
using System.Text;
using System.Threading.Tasks;
using Microsoft.AspNetCore.Mvc;
using Microsoft.Azure.WebJobs;
using Microsoft.Azure.WebJobs.Extensions.Http;
using Microsoft.AspNetCore.Http;
using Microsoft.Extensions.Logging;
using Newtonsoft.Json.Linq;

namespace dotnet_custom
{
    public static class custom
    {
        [FunctionName("custom")]
        public static async Task<HttpResponseMessage> Run(
            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = null)] HttpRequest req,
            ILogger log)
        {   
            Stopwatch sw = new Stopwatch();
            sw.Start();

            /* 
            TODO: put your code here
            You can basically do anything you want,
            but please leave the return statement header
            and the success field as it is.
            */

            sw.Stop();
            JObject message = new JObject();
            message.Add("success", new JValue(true));
            JObject payload = new JObject();
            payload.Add("test", new JValue("custom test"));
            payload.Add("time", new JValue(sw.Elapsed.TotalMilliseconds));
            message.Add("payload", payload);

            return new HttpResponseMessage(HttpStatusCode.OK) {
                Content = new StringContent(message.ToString(), Encoding.UTF8, "application/json")
            };
        }
    }
}
